src/scheduler/server_info.c

```diff
@@ -223,8 +223,12 @@ query_server(status *pol, int pbs_sd)
 		return NULL;
 	}
 
+	for(sched = pbs_statsched(pbs_sd, NULL, NULL);
+			((sched != NULL) && strcmp(sc_name, sched->name));
+			sched = sched->next
+		);	//empty for loop
```

Comment:
Same here, use /* */

Reply:
>Same here, use /* */

The comment style is changed to /* */ in the second commit